pyuvdata/uvcal.py

```diff
@@ -432,6 +432,28 @@ def select(self, antenna_nums=None, antenna_names=None,
         if run_check:
             self.check(run_check_acceptability=run_check_acceptability)
 
+    def convert_to_gain(self):
+        if self.cal_type == 'gain':
+            raise ValueError('The data is already a gain cal_type.')
+        elif self.cal_type == 'delay':
+            phase_array = np.zeros((self.Nants_data, self.Nspws, self.Nfreqs, self.Ntimes, self.Njones))
+            for si in range(self.Nspws):
+                temp = -2 * np.pi * np.dot(self.delay_array[:, si, :, :, np.newaxis],
```

Comment:
That's a very good point. I did this based off of what firstcal does, since I think it's the only thing using the delay type, but I can add a delay convention parameter to the method. Or were you thinking it should be on the object?